orc8r/cloud/go/services/state/indexer/reindex/reindexer_singleton.go

```diff
@@ -48,21 +48,33 @@ func (r *reindexerSingleton) Run(ctx context.Context) {
 	// TODO(reginawang3495) will support non-nil sendUpdate from Run after removing queue impl
 	var sendUpdate func(string) = nil
 
-	batches := r.getReindexBatches(ctx)
 	for {
```

Comment:
For infinite loops, a good pattern is: 1. an outer func that creates the infinite loop, then, 2. a single func called within the loop. Makes validation and conceptualization easier

So in that case, it would be: let's make inside the for loop just be 3 things

1. is canceled
2. reindex jobs
3. log any err from reindex jobs

This way, reindexJobs will handle creating its own jobs, rather than taking it as an arg